### PR TITLE
feat(avatars): enable default avatar

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 2fff770f2a235cf80786dd9cb7d4ceb9612de6a9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 33f3d9998083e7a4731cf0a9e0c3f336dfe5733e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-profile-server

---
This message was auto-generated by https://gitcop.com

File: lib/routes/avatar/delete.js [hunk 1]
```diff
@@ -30,6 +32,11 @@ module.exports = {
     }
   },
   handler: function deleteAvatar(req, reply) {
+    if (req.params.id === DEFAULT_AVATAR_ID) {
```
Comment (vladikoff) [inline lib/routes/avatar/delete.js]: this is new, to prevent errors on `Clear` avatar

File: lib/routes/profile.js [hunk 2]
```diff
@@ -65,6 +70,11 @@ module.exports = {
         if (creds.scope.indexOf('openid') !== -1) {
           result.sub = creds.user;
         }
+
+        if (result.avatar) {
+          result.avatarDefault = result.avatar === DEFAULT_AVATAR_URL;
```
Comment (vladikoff) [inline lib/routes/profile.js]: this helps keep the batch requests clean (currently they extract a single property and is too much to refactor, but we can in the future)

Comment (rfk) [reply lib/routes/profile.js]: Sounds good; might be worth leaving that comment in the code for our future selves as well.

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 2fff770f2a235cf80786dd9cb7d4ceb9612de6a9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 33f3d9998083e7a4731cf0a9e0c3f336dfe5733e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-profile-server

---
This message was auto-generated by https://gitcop.com

**Pull request merged**